### PR TITLE
Implement highlighting and tier overlay for emerald pouch items

### PR DESCRIPTION
**Pull request by ekgame**
Implements emerald pouch item highlighting and tier overlay.

<table>
<tr>
	<td>Before</td>
	<td>After</td>
<tr>
	<td>
	<td>
</table>

This pull request also includes some related improvements:
- Improve roman numeral parsing
- Do not emit an "invalid item" error if emerald pouch item does not have a usage line. Valid emerald pouch items in the market do not have the line. It should be safe to assume that even a truly invalid item simply does not contain emerald.

### CODE REVIEW

File: common/src/main/java/com/wynntils/utils/MathUtils.java [hunk 1]
```diff
@@ -4,9 +4,24 @@
  */
 package com.wynntils.utils;
 
+import java.util.HashMap;
 import java.util.Locale;
+import java.util.Map;
 
 public final class MathUtils {
+
+    private static final Map<Integer,Integer> romanNumeralsMap = new HashMap<>();
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/utils/MathUtils.java]: TreeMap is better for this?

File: common/src/main/java/com/wynntils/wynn/item/parsers/EmeraldPouchParser.java [hunk 2]
```diff
@@ -22,13 +22,7 @@ public static int getPouchUsage(ItemStack stack) {
...
         Matcher usageMatcher = POUCH_USAGE_PATTERN.matcher(lore);
         if (!usageMatcher.find()) {
-            if (lore.contains("§7Empty")) {
-                return 0;
-            }
-
-            WynntilsMod.error(
-                    "EmeraldPouchParser#getPouchUsage was called on an ItemStack that wasn't an emerald pouch");
-            return -1;
+            return 0;
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/wynn/item/parsers/EmeraldPouchParser.java:25]: This fixes inventory emerald count too. Thanks

File: common/src/main/java/com/wynntils/utils/MathUtils.java (hunk 1 above)
Comment (ekgame) [reply common/src/main/java/com/wynntils/utils/MathUtils.java]: I don't really see why TreeMap would be more appropriate. The map only has 7 items, so memory usage and the order of entries is not an issue.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/utils/MathUtils.java]: Sure, fine either way.

**Review by kristofbolyai — DISMISSED** [review]

**Comment by kristofbolyai** [comment]
@ekgame 
```
Execution failed for task ':common:spotlessJava'.
> Do not use wildcard imports.  'spotlessApply' cannot resolve this issue.
```

Could you remove wildcard imports? Otherwise this looks good!

**Comment by ekgame** [comment]
Looks like my IDE collapsed them automatically, should be good now.

**Review by kristofbolyai — DISMISSED** [review]

**Review by kristofbolyai — APPROVED** [review]

**Comment by magicus** [comment]
Thank you for your contribution to Artemis @ekgame!

**Comment by ekgame** [comment]
> Thank you for your contribution to Artemis @ekgame!

Happy to help!

**Pull request merged**